spec/requests/v6/assessments_controller/realistic_civil_apply_spec.rb

```diff
@@ -0,0 +1,244 @@
+            "payments": [
+              {
+                "amount": 8000.0,
+                "frequency": "annual",
+                "income_type": "student_loan",
+              },
+            ],
+          },
+          "other_incomes": [
+            {
+              "source": "Maintenance in",
+              "payments": [
+                {
+                  "date": "2023-04-26",
+                  "amount": 167.0,
+                  "client_id": "3000-01-01",
+                },
+              ],
+            },
+          ],
+          "outgoings": [
+            {
+              "name": "rent_or_mortgage",
+              "payments": [
+                {
+                  "amount": 563.08,
+                  "client_id": "3000-01-01",
+                  "payment_date": "2023-05-02",
+                  "housing_cost_type": "mortgage",
+                },
+              ],
+            },
+            {
+              "name": "maintenance_out",
+              "payments": [
+                {
+                  "amount": 30.0,
+                  "client_id": "3000-01-01",
+                  "payment_date": "2023-05-15",
+                },
+              ],
+            },
+          ],
+          "properties": {
+            "main_home": {
+              "value": 720_000.0,
+              "percentage_owned": 50.0,
+              "outstanding_mortgage": 325_000.0,
+              "shared_with_housing_assoc": false,
+            },
+            "additional_properties": [
+              {
+                "value": 435_000.0,
+                "percentage_owned": 100.0,
+                "outstanding_mortgage": 435_000.0,
+                "shared_with_housing_assoc": false,
+              },
+            ],
+          },
+          "regular_transactions": [
+            {
+              "amount": 90.0,
+              "category": "benefits",
+              "frequency": "four_weekly",
+              "operation": "credit",
+            },
+          ],
+          "state_benefits": [
+            {
+              "name": "personal_independent_payments",
+              "payments": [
+                {
+                  "date": "2023-05-26",
+                  "amount": 691.0,
+                  "client_id": "3000-01-01",
+                },
+                {
+                  "date": "2023-06-26",
+                  "amount": 691.0,
+                  "client_id": "3000-01-01",
+                },
+                {
+                  "date": "2023-07-24",
+                  "amount": 691.0,
+                  "client_id": "3000-01-01",
+                },
+              ],
+            },
+          ],
+          "vehicles": [
+            {
+              "value": 3500.0,
+              "in_regular_use": true,
+              "date_of_purchase": "2021-07-19",
+              "loan_amount_outstanding": 1500.0,
+            },
+          ],
+          "explicit_remarks": [
+            {
+              "details": %w[
```

Comment:
`%w`?

Reply:
Ruby shorthand for an array of strings: `%w[foo bar]` is the same as `["foo", "bar"]` and the former is the preferred way of expressing it according to the GDS "rubocop" linting rules.

Reply:
ah, makes sense to keep it like this then 👍

Reply:
rubocop converts array of strings to this format